### DISCUSSION TITLE
What information does the git commit contains?

### DISCUSSION THREAD

**Discussion by JaakkoJSeppala** [community/community#23252] [New to GitHub]
<p>In git, what information does the git commit contains</p>
<p>SHA1<br>
SSH-key<br>
Message<br>
Changes made in previous commit<br>
A reference to previous commit<br>
Author of the commit</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can display all of it using git show --format=raw &lt;commit&gt;. Note that this only affects the commit data itself, the diff is shown as usual. For details take a look at the <a href="https://git-scm.com/docs/git-show#_pretty_formats" rel="nofollow noopener">git show documentation section on “pretty formats”</a>.</p>
